ext_tests.py

```diff
@@ -86,7 +86,7 @@ def add_test_method(cls, name, test):
 
 
 
-modules = ['jsonpatch']
+modules = ['jsonpatch']F
```

Comment:
Not quite sure what happened here. Can you please remote that? Thanks :)

Reply:
thx - fixed